docs/usage/quickstart.rst

```diff
@@ -107,6 +113,9 @@ you can set either values or arrays at a time:
     hist[bh.underflow] = 0 # set the underflow bin
     hist2d[3:5, 2:4] = np.eye(2) # set with array
 
+For non-simple storages, you can add an extra dimension that matches the
+constructor arguments of that accumulator.
```

Comment:
Can you add an example? I am not sure what you mean.

Reply:
Added one.